src/sgame/sg_bot.cpp

```diff
@@ -202,21 +202,11 @@ const char * G_BotGetBehavior( int clientNum )
 	return bot->botMind->behaviorTree->name;
 }
 
-void G_BotChangeBehavior( int clientNum, const char* behavior )
+bool G_BotSetBehavior( int clientNum, const char* behavior )
 {
-	gentity_t *bot = &g_entities[clientNum];
-
-	if ( !( bot->r.svFlags & SVF_BOT ) || !bot->botMind )
```

Comment:
The check is needed because the `bot behavior` subcommand can be invoked on random client nums.

Reply:
Fair enough, dropped that commit